src/resolve/resolved-varlink.c

```diff
@@ -1228,6 +1251,43 @@ static int vl_method_resolve_record(Varlink *link, JsonVariant *parameters, Varl
         return 1;
 }
 
+static int vl_method_start_browse(Varlink* link, JsonVariant* parameters, VarlinkMethodFlags flags, void* userdata) {
+        static const JsonDispatch dispatch_table[] = {
+                { "domainName", JSON_VARIANT_STRING,   json_dispatch_string, offsetof(LookupParametersMdnsBrowse, domainName), JSON_MANDATORY },
+                { "name",       JSON_VARIANT_STRING,   json_dispatch_string, offsetof(LookupParametersMdnsBrowse, name),       JSON_MANDATORY },
+                { "type",       JSON_VARIANT_STRING,   json_dispatch_string, offsetof(LookupParametersMdnsBrowse, type),       JSON_MANDATORY },
+                { "ifindex",    JSON_VARIANT_INTEGER,  json_dispatch_int,    offsetof(LookupParametersMdnsBrowse, ifindex),    JSON_MANDATORY },
```

Comment:
I wonder if "ifindex" should be mandatory? The other methods default to 0 where it means "all the interfaces". I tried passing 0 explicitly too but it was rejected with
```sh
Method call failed: org.varlink.service.InvalidParameter
{
        "parameter" : "ifindex"
}
```